src/main/webapp/app/exam/manage/exams/exam-update.component.ts

```diff
@@ -159,6 +229,13 @@ export class ExamUpdateComponent implements OnInit {
         this.isSaving = false;
     }
 
+    /**
+     * Returns true if the original exam is currently ongoing before any changes, false otherwise.
+     */
+    get isOngoingExam(): boolean {
+        return !!this.exam.id && !!this.originalStartDate && !!this.originalEndDate && dayjs().isBetween(this.originalStartDate, this.originalEndDate);
```

Comment:
```suggestion
        return !!(this.exam.id && this.originalStartDate && this.originalEndDate && dayjs().isBetween(this.originalStartDate, this.originalEndDate));
```

Reply:
Isnt it possible to just leave out these double `!!` at all? You use these to "cast" the objects to booleans, right? But through the `&&` operator, js should just take the type of the last one, which is already a boolean.

![image](https://github.com/ls1intum/Artemis/assets/78978542/227fc68a-d2c7-44ae-9b46-d34673663b0a)

Reply:
Nope

<img width="510" alt="Screenshot 2023-10-10 at 16 03 15" src="https://github.com/ls1intum/Artemis/assets/1051396/109e0164-f982-41e1-ab41-89278d40202f">

Reply:
Probably in this special case, as we know, that the exam id is never `0` due to application constraints

Reply:
But even if `num` is `1`, as soon as `date` is undefined, the expression is `undefined`. Therefore the need for `!!`.

<img width="453" alt="Screenshot 2023-10-10 at 16 54 39" src="https://github.com/ls1intum/Artemis/assets/1051396/8cf94ead-f126-4d0c-aa66-78a5aa561bc7">